apps/super-app/src/common/components/navbar/NavIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from "react-router-dom";
+function NavIcon(props:any) {
```

Comment:
Please use a more suitable type, [personal-calls/frontend/src/common/components/Container.tsx](https://github.com/bettersg/call-home/pull/102/files#diff-931083105ec009d12e6eaa8b9295fbac95aeb33cdcde68414079b8af1bbfafa8) has an example. You can also rely on the CI check to warn on this https://github.com/bettersg/call-home/actions/runs/3724490539/jobs/6316677421.